src/main/java/pl/javastart/task/Book.java

```diff
@@ -0,0 +1,13 @@
+package pl.javastart.task;
+
+public class Book {
+    String title;
+    String author;
+    String publisher;
+
+    public Book(String t, String a, String p){
```

Comment:
Checkstyle zauważył w tej linii błąd. To, co znalazł możesz podejrzeć w pull requeście z zakładce "Files changed". Polecam zainstalować sobie dodatek do intellij, żeby na bieżąco informował. Instrukcję znajdziesz tutaj https://bootcamp.javastart.pl/strona/jakosc-kodu

W tym przypadku brakuje spacji przed `{`